src/elements/header/spec/script.spec.js

```diff
@@ -0,0 +1,5 @@
+import header from '../script';
+
```

Comment:
I like that you create a special folder for that because of the many files it could generate but is spec/script.spec.js best standard?
What do other projects do?
is spec/index.js better?